source/style/style.css

```diff
@@ -0,0 +1,581 @@
+  margin-right: 0.2rem;
+}
+
+.workout-time i.fa-clock {
+  margin-right: 0.6rem;
+  font-size: 1rem;
+}
+
+/* Services Section */
+.services {
+  padding-right: 20px;
+  padding-left: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-image: url("../images/services.webp");
+  background-position: center();
+  background-size: cover;
+}
+
+.services .services-header {
+  margin: 1.5rem auto 2rem;
+  font-size: 1.8rem;
+  color: var(--gray);
+  padding-bottom: 1rem;
+  position: relative;
+  font-family: lg;
+}
+
+.services .services-header::after {
+  content: " ";
+  width: 20%;
+  display: block;
+  height: 1px;
+  position: absolute;
+  background-color: var(--orange);
+  margin: 1rem auto;
+  left: 40%;
+  right: 40%;
+}
+
+.card {
+  background-color: rgba(247, 230, 230, 0.1);
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  margin-bottom: 1rem;
+  border: 1px solid transparent;
+  transition: all 0.5s;
+}
+
+.card:hover {
+  border: 1px solid var(--gray);
+  transform: scale(1.1, 1.1);
+}
+
+.card .card-image i {
+  font-size: 3rem;
+  color: var(--gray);
+}
+
+.card-header {
+  color: var(--orange);
+  margin: 1rem 1rem;
+}
+
+.card-body p {
+  color: var(--gray);
+  font-family: sm;
+}
+
+.btn-join {
+  background-color: var(--orange);
+  padding: 1rem;
+  text-align: center;
+  text-decoration: none;
+  font-size: 1.2rem;
+  color: var(--gray);
+  margin-bottom: 3rem;
+  transition: all 0.3s;
+}
+
+.btn-join:hover {
+  background-color: #ac3f33;
+}
+
+/* Feature speaker */
+.feature-speaker {
+  background-image: none;
+}
+
+.change-height {
+  height: 250px;
+  overflow: hidden;
+  transition: all 0.5s;
+}
+
+.feature-card {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  margin-bottom: 3rem;
+  width: 100%;
+}
+
+.feature-card .feature-card-image {
+  width: 60%;
+  margin-right: 2rem;
+}
+
+.feature-card .feature-card-image img {
+  width: 100%;
+  height: 100px;
+  object-fit: cover;
+}
+
+.body-builder-image {
+  position: relative;
+  top: -40%;
+  left: 25%;
+}
+
+.feature-card-body h6 {
+  font-size: 1.2rem;
+  font-family: lg;
+  color: var(--black);
+  margin-bottom: 1rem;
+  position: relative;
+}
+
+.feature-card-body h6::after {
+  content: "";
+  width: 50px;
+  height: 2px;
+  display: block;
+  position: absolute;
+  top: 100%;
+  background-color: var(--gray);
+  margin-top: 0.5rem;
+}
+
+#feature-card-job {
+  display: block;
+  margin-top: 0.2rem;
+  color: var(--orange);
+  font-size: 0.8rem;
+  font-style: italic;
+  font-family: lg;
+}
+
+.feature-card-body p {
+  font-size: 1rem;
+  font-family: md;
+}
+
+button.btn-join {
+  background-color: #fff;
+  border: 1px solid var(--orange);
+  outline: none;
+  margin-top: 1rem;
+}
+
+/* Partner */
+.partner {
+  background-color: rgba(20, 20, 20, 0.6);
+  display: none;
+}
```

Comment:
- [ ] Kindly use `background-color: #414246`. This looks more like the given design.
- [ ] [Optional] You can make this section look better by adding some padding.

![image](https://user-images.githubusercontent.com/92215878/190709212-0f239a13-89b9-4f95-8454-d15f860686f9.png)